registration-processor/pre-processor/registration-processor-osi-validator-stage/src/main/java/io/mosip/registration/processor/stages/config/OSIConfigBean.java

```diff
@@ -13,34 +13,15 @@
 import io.mosip.kernel.crypto.jce.core.CryptoCore;
 import io.mosip.kernel.keygenerator.bouncycastle.KeyGenerator;
 import io.mosip.registration.processor.packet.storage.utils.AuthUtil;
-
-import io.mosip.registration.processor.stages.osivalidator.OSIValidator;
-import io.mosip.registration.processor.stages.osivalidator.OSIValidatorStage;
-import io.mosip.registration.processor.stages.osivalidator.UMCValidator;
 import io.mosip.registration.processor.stages.osivalidator.utils.OSIUtils;
 
 @Configuration
 public class OSIConfigBean {
 
-	@Bean

```

Comment:
Since the OSIValidator, UMCValidator and OSIUtils are already being service annotated with @Service, their beans are automatically created, and hence having their bean definition additionally resulted in error in spring bean loading. So removed the bean definitions.

Reply:
Before that was not the case rt, why is bean taking the @service now with the new changes ?